connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoReader.scala

```diff
@@ -69,6 +72,48 @@ private[kusto] object KustoReader {
                                           options: KustoReadOptions,
                                           filtering: KustoFiltering): RDD[Row] = {
     KDSU.logInfo(myName, s"Starting exporting data from Kusto to blob storage. requestId: ${request.requestId}")
+    var paths: Seq[String] = Seq()
+    // if exportQueryOnlyOnce is set to true, then check if path is cached and use it
+    // if not export and cache the path for reuse later
+    if (options.exportQueryOnlyOnce) {
```

Comment:
we should log that we are using the cache and weather or not did we get a cache hit